engine/execution/rpc/engine.go

```diff
@@ -545,7 +545,7 @@ func (h *handler) GetAccountAtBlockID(
 
 	value, err := h.engine.GetAccount(ctx, flowAddress, blockFlowID)
 	if err != nil {
-		if errors.Is(err, scripts.ErrStateCommitmentPruned) {
+		if errors.Is(err, state.ErrStateCommitmentPruned) {
 			return nil, status.Errorf(codes.OutOfRange, "state for block ID %s not available", blockFlowID)
 		}
 		if errors.Is(err, storage.ErrNotFound) {
```

Comment:
I think this is the case that block is not found, rather than account not found, might need to update error message